consensus/service/src/api/peer_api_service.rs

```diff
@@ -252,7 +252,12 @@ impl ConsensusPeerApi for PeerApiService {
                     }
 
                     Err(peer_service_error) => match peer_service_error {
-                        PeerServiceError::Enclave(err) => Err(rpc_enclave_err(err, logger)),
+                        PeerServiceError::Enclave(err) => match err {
```

Comment:
I think this can be simplified into a single match statement:
```
Err(PeerServiceError::Enclave(Error::Attest(_))) => rpc_permissions_error("peer_tx_propose", err, logger)),
Err(err) => rpc_internal_error("peer_tx_propose", err, logger)
```